api/profile/profileRouter.js

```diff
@@ -167,7 +167,7 @@ router.post('/', authRequired, async (req, res) => {
       await Profiles.findById(id).then(async (pf) => {
         if (pf == undefined) {
           //profile not found so lets insert it
-          await Profiles.create(profile).then((profile) =>
+          await Profiles.create(profile).then((profile) =>//naming convention duplicated
```

Comment:
We should remove this comment as there is no reason to change this file in this PR

Reply:
Comment removed